services/core-web/src/components/common/FileUpload.js

```diff
@@ -53,7 +53,7 @@ class FileUpload extends React.Component {
           chunkSize: this.props.chunkSize,
           metadata: {
             filename: file.name,
-            filetype: file.type,
+            filetype: file.type || "application/octet-stream",
```

Comment:
May be "application/octet-stream" can be moved to constants. What do you think?

Reply:
Not sure what the heck happened but it's resorted back to not working--so I'll have to take another look. :/